code/modules/wiremod/components/action/radio.dm

```diff
@@ -65,8 +74,10 @@
 	INVOKE_ASYNC(src, PROC_REF(handle_radio_input), port)
 
 /obj/item/circuit_component/radio/proc/handle_radio_input(datum/port/input/port)
-	var/frequency = freq.value
+	if(!TIMER_COOLDOWN_CHECK(src, COOLDOWN_SIGNALLER_SEND))
+		return
 
```

Comment:
this needs to be on the parent aka the integrated circuit, not the component itself
Otherwise you can stack 50 components on the same circuit and accomplish a cooldown bypass.